src/piper_cmd2_name_lexer.xrl

```diff
@@ -0,0 +1,20 @@
+Definitions.
+
+NAME                        = [a-z][a-zA-Z0-9_\-]*
+HIPCHAT_EMOJI               = \([a-zA-Z0-9_\-\+]+\)
+SLACK_EMOJI                 = :[a-zA-Z0-9_\-\+]+:
```

Comment:
I think that's a decision for a different time since removing emoji support is a subtractive change. Open to the idea, though, since I don't think it's gotten much use nor have we publicized the capability.